src/util.js

```diff
@@ -235,7 +235,7 @@ function isFirstChildrenInlineNode(path) {
  * appear as function arguments or array elements
  */
 function docShouldHaveTrailingNewline(path) {
-  return ["entry"].includes(path.getParentNode().kind);
+  return ["array"].includes(path.getParentNode().kind);
```

Comment:
`return path.getParentNode().kind === "array"`

Reply:
@czosel in next parser update we will have not only `array` (i know :smile: ), let's left this as is here :+1: